apiserver/utility/utility_test.go

```diff
@@ -114,15 +114,6 @@ workflows:
 		require.True(t, strings.Contains(err.Error(), "Validation failed: Config validation error: Failed to get config (bitrise.yml) from base 64 data, err: Failed to parse bitrise config, error: missing format_version"), err.Error())
 	}
 
-	t.Log("Invalid secrets - empty")
```

Comment:
Could add a positive test case, instead of removing it completely.

Reply:
See my latest commit please. There was a test for valid cases, I extended the list.